microservices/Keli-Katheya-User-Registration/src/main/java/com/kelikatheya/microservices/userRegistration/KeliKatheyaUserRegistrationApplication.java

```diff
@@ -0,0 +1,22 @@
+package com.kelikatheya.microservices.userRegistration;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@SpringBootApplication(scanBasePackages={
+		"com.kelikatheya.microservices"})
```

Comment:
Do we need scanBasePackages ?

Reply:
If I remove its throwing exception, failed to create ServletWebServerFactory bean